database/sqlite.go

```diff
@@ -87,31 +87,29 @@ func sqliteConnection(logger log.Logger, path string) *sqlite {
 // TestSQLiteDB is a wrapper around sql.DB for SQLite connections designed for tests to provide
 // a clean database for each testcase.  Callers should cleanup with Close() when finished.
 type TestSQLiteDB struct {
-	DB *sql.DB
-
-	dir string // temp dir created for sqlite files
-
+	*sql.DB
```

Comment:
Embedded this to avoid having to use `db.DB` when we want to pass on the db connection. In almost all cases where we're using this wrapper struct, we pass on the underlying `sql.DB` value.

```go
	db := database.CreateTestSqliteDB(t)
	repo := NewRepo(log.NewNopLogger(), db.DB)
         // turns into
	db := database.CreateTestSqliteDB(t)
	repo := NewRepo(log.NewNopLogger(), db)
```